installer/lib/mix/tasks/local.phx.ex

```diff
@@ -1,17 +1,21 @@
 defmodule Mix.Tasks.Local.Phx do
   use Mix.Task
 
-  @url "https://github.com/phoenixframework/archives/raw/master/phx_new.ez"
-  @shortdoc "Updates the Phoenix project generator locally"
+  @url "https://github.com/phoenixframework/archives/raw/master/phx_new-1.3.4.ez"
+  @shortdoc "Deprecated task to update the Phoenix project generator locally"
 
   @moduledoc """
-  Updates the Phoenix project generator locally.
+  This task is deprecated. To update the Phoenix project generator, use hex:
 
-      mix local.phx
-
-  Accepts the same command line options as `archive.install`.
+      mix archive.install hex phx_new
   """
+  @deprecated ~s'Use "mix archive.install hex phx_new" instead'
   def run(args) do
-    Mix.Task.run "archive.install", [@url | args]
+    IO.puts(
+      :stderr,
+      "WARNING: mix local.phx is no longer supported, use: mix archive.install hex phx_new"
+    )
+
+    Mix.Task.run("archive.install", [@url | args])
```

Comment:
Hi @adamu! Thanks for the PR and great job answering on Stack Overflow. Instead of deprecating it, I think we should simply update it to use Hex. Something like this:

```suggestion
    Mix.Task.run("archive.install", ["hex", "phx_new" | args])
```

Can you please remove the warning and update it?